src/com/pentalog/pentastagiu/homework2/shop/ClientCart.java

```diff
@@ -0,0 +1,25 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ClientCart {
+
+    private ClientsOfShop name;
+    ItemsOnSale price;
```

Comment:
nu imi este f clar cum ti-ai imaginat logica de aici, dar mai potrivit ar fi fost sa ai o lista de produse, initial goala, si o metoda care sa iti permita sa adaugi rand pe rand cate un produs in cos
apoi aveai 2 optiuni: 
1. tot in metoda in care adaugai un produs, sa adaugi pretul produsului adaugat la valoarea totala a cosului 
2. o metoda de calculare a valorii totale a cosului in care luai in calcul pretul produselor din cos(iterai pe produsele din cos si adaugai pe rand pretul fiecarui produs) pe care aplicai un discount in functie de tipul de membership al clientului